src/modules/lifecycle_management.json

```diff
@@ -326,6 +496,91 @@
                         "$ref": "#/components/schemas/LifecycleFinishedParameters"
                     }
                 }
+            },
+            "AppSession": {
+                "title": "AppSession",
+                "type": "object",
+                "required": [
+                    "app",
+                    "runtime"
+                ],
+                "properties": {
+                    "app": {
+                        "description": "Information about the application",
+                        "type": "object",
+                        "required": [
+                            "id",
+                            "url"
```

Comment:
Is URL required for all runtimes?

Reply:
yeah, I was debating on this. We are using the url as a way to generate a unique hash for the device.uid. I guess I could make this not required, but we would need a fallback plan if it is not passed and then the app calls Device.uid

Reply:
okay, seems fine, then.